deb/debian/stackdriver-agent.init

```diff
@@ -27,9 +27,10 @@ export PATH=/sbin:/bin:/usr/sbin:/usr/bin
 DESC="Stackdriver metrics collection agent"
 NAME=stackdriver-agent
 DAEMON=/opt/stackdriver/collectd/sbin/stackdriver-collectd
+COLLECTDMON=/opt/stackdriver/collectd/sbin/stackdriver-collectdmon
 
 CONFIG=/etc/stackdriver/collectd.conf
-_PIDFILE=/var/run/stackdriver-agent.pid
```

Comment:
The service name is still `stackdriver-agent`, so we should keep the original PID file name, IMO.

Reply:
After turning on collectdmon, there is no stackdriver-agent.pid but only collectdmon.pid file. 

![image](https://user-images.githubusercontent.com/8354694/98711867-8ffe5680-2353-11eb-921d-80f9f4c90063.png)

Reply:
COLLECTDMON_PIDFILE  is defined here: https://github.com/Stackdriver/collectd/blob/stackdriver-agent-5.8.1/src/collectdmon.c#L67

Reply:
COLLECTDMON_PIDFILE is the default value, which will get overridden by the `-P` option.
What I'm saying is that the service is called `stackdriver-agent`, so the pid file for the main service process should stay `stackdriver-agent.pid`, even if the process itself is different.
Also in `rpm/stackdriver-agent`.

Reply:
Yes, you are right
![image](https://user-images.githubusercontent.com/8354694/98728128-0c4e6500-2367-11eb-8090-2068936b81b6.png)

https://github.com/Stackdriver/collectd/blob/stackdriver-agent-5.8.1/src/collectdmon.pod